framework/include/vectorpostprocessors/LineMaterialSamplerBase.h

```diff
@@ -179,6 +184,7 @@ LineMaterialSamplerBase<T>::execute()
       addSample(_q_point[qp], qp_proj_dist_along_line, values);
     }
   }
+  _fe_problem.clearActiveMaterialProperties(_tid);
```

Comment:
The setup and teardown in the compute threads classes occurs once per looping over the mesh - not a problem.  But the one here, for example causes the prop dependency setup/teardown to occur for every element this postprocessor executes on.